sippy-ng/src/component_readiness/CompReadyTestReport.js

```diff
@@ -186,20 +184,21 @@ export default function CompReadyTestReport(props) {
         <div style={{ display: 'block' }}>Environment: {environment}</div>
         <br />
         <div style={{ display: 'block' }}>
-          Fisher Exact: {data.fisher_exact.toFixed(4)}
+          {/* data.fisher_exact is from 0-1; from that we calculate the probability of regression
+              expressed as a percentage */}
+          <Tooltip
+            title="Test results for individual Prow Jobs may not be statistically
+          significant, but when taken in aggregate, there may be a statistically
+          significant difference compared to the historical basis"
+          >
+            <InfoIcon />
+          </Tooltip>
+          Probability of regression:{' '}
+          {((1.0 - data.fisher_exact) * 100.0).toFixed(2)}%
```

Comment:
When I click on a green square I get "Probability of regression: 100.00%"

e.g. http://localhost:3000/sippy-ng/component_readiness/test_details?arch=amd64&baseEndTime=2023-06-01%2023%3A59%3A59&baseRelease=4.13&baseStartTime=2023-05-02%2000%3A00%3A00&capability=Builds&component=Build&confidence=95&environment=ovn%20no-upgrade%20amd64%20alibaba%20standard&groupBy=cloud%2Carch%2Cnetwork&ignoreDisruption=true&ignoreMissing=false&minFail=3&network=ovn&pity=5&platform=alibaba&sampleEndTime=2023-06-01%2023%3A59%3A59&sampleRelease=4.14&sampleStartTime=2023-05-02%2000%3A00%3A00&testId=openshift-tests%3A1e14ddbe1fdda1ccf10813c07e0a16c5&testName=%5Bsig-builds%5D%5BFeature%3ABuilds%5D%20build%20have%20source%20revision%20metadata%20started%20build%20should%20contain%20source%20revision%20information%20%5Bapigroup%3Abuild.openshift.io%5D%20%5BSkipped%3ADisconnected%5D%20%5BSuite%3Aopenshift%2Fconformance%2Fparallel%5D&upgrade=no-upgrade&variant=standard

And a red square gives me "Probability of regression: 99.39%"

http://localhost:3000/sippy-ng/component_readiness/test_details?arch=amd64&baseEndTime=2023-06-01%2023%3A59%3A59&baseRelease=4.13&baseStartTime=2023-05-02%2000%3A00%3A00&capability=Builds&component=Build&confidence=95&environment=ovn%20no-upgrade%20amd64%20alibaba%20standard&groupBy=cloud%2Carch%2Cnetwork&ignoreDisruption=true&ignoreMissing=false&minFail=3&network=ovn&pity=5&platform=alibaba&sampleEndTime=2023-06-01%2023%3A59%3A59&sampleRelease=4.14&sampleStartTime=2023-05-02%2000%3A00%3A00&testId=openshift-tests%3Acbdb82eb4068ba3dc50dee9bc862a06d&testName=%5Bsig-builds%5D%5BFeature%3ABuilds%5D%20Optimized%20image%20builds%20should%20succeed%20%5Bapigroup%3Abuild.openshift.io%5D%20%5BSkipped%3ADisconnected%5D%20%5BSuite%3Aopenshift%2Fconformance%2Fparallel%5D&upgrade=no-upgrade&variant=standard



Can fix in a follow-up, I want the rest of these changes